### PR TITLE
Move "common keys" section of plugin docs to top of the page

### PR DESCRIPTION
**Pull request by happz**
This is shared among all plugins of a step, and it's not necessary to repeat it for every plugin.

Pull Request Checklist

* [x] implement the feature

### CODE REVIEW

**Review by therazix — APPROVED** [review]

**Comment by thrix** [comment]
@happz there is a typo in the commit message `plugn`

File: docs/templates/plugins.rst.j2 [hunk 1]
```diff
@@ -61,30 +31,68 @@ Configuration
...
+{{ '~' * (8 + (STEP | length)) }}
 
-{% if inherited_fields %}
-Common Keys
-"""""""""""
+{% for PLUGIN_ID, PLUGIN, PLUGIN_DATA_CLASS in PLUGINS() %}
+    {% if loop.first %}
+.. _plugins/{{ STEP }}/common-keys:
+
+Common keys
```
Comment (thrix) [inline docs/templates/plugins.rst.j2:49]:
```suggestion
Common Keys
```

so it better matches the documentation style. I see titles use Pascal Case

File: docs/templates/plugins.rst.j2 [hunk 2]
```diff
@@ -61,30 +31,68 @@ Configuration
...
+{{ PLUGIN.__doc__ | dedent | trim }}
+{% endif %}
+
+{% set intrinsic_fields = container_intrinsic_fields(PLUGIN_DATA_CLASS) | sort %}
+
+{% if intrinsic_fields %}
+Configuration
+^^^^^^^^^^^^^
+
+See also :ref:`Common keys<plugins/{{ STEP }}/common-keys>` accepted by the plugin.
```
Comment (thrix) [inline docs/templates/plugins.rst.j2:88]:
```suggestion
See also :ref:`Common Keys<plugins/{{ STEP }}/common-keys>` accepted by the plugin.
```

**Review by thrix — APPROVED** [review]
LGTM

File: docs/templates/plugins.rst.j2 (hunk 1 above)
Comment (happz) [reply docs/templates/plugins.rst.j2]: Fixed in dad40531

File: docs/templates/plugins.rst.j2 (hunk 2 above)
Comment (happz) [reply docs/templates/plugins.rst.j2]: Fixed in dad40531

**Review by martinhoyer — APPROVED** [review]

**Review by thrix — APPROVED** [review]
LGTM

**Comment by happz** [comment]
Documentation-only, merging.

**Pull request merged**